lib/add-tests.coffee

```diff
@@ -1,5 +1,5 @@
 async = require 'async'
-_ = require 'underscore'
+_ = require 'lodash'
```

Comment:
Replacement everywhere is longterm goal. But it's not exactly apples to apples. The lodash replacements have nuances (e.g., one deep copies when the other shallow copies), so I only replaced here where I could verify outcome.